### PR TITLE
fix(datepicker): fix 'util' import and use 'isInteger'

### PR DESCRIPTION
**Pull request by maxokorokov**

### CODE REVIEW

File: src/datepicker/adapters/ngb-date-adapter.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import {Injectable} from '@angular/core';
 import {NgbDateStruct} from '../ngb-date-struct';
-import {isNumber} from 'util';
+import {isInteger} from '../../util/util';
```
Comment (maxokorokov) [inline src/datepicker/adapters/ngb-date-adapter.ts]: `isNumber` from `util` actually corresponds to `isInteger` from our `../../util/util`...

**Comment by maxokorokov** [comment]
Wait, don't merge

**Comment by maxokorokov** [comment]
Ok now, sorry

**Comment by pkozlowski-opensource** [comment]
LGTM, thnx!

**Pull request merged**